docs/book/src/reference/platform.md

```diff
@@ -0,0 +1,240 @@
+            "os": "linux"
+         }
+      },
+      {
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "size": 739,
+         "digest": "sha256:621288f6573c012d7cf6642f6d9ab20dbaa35de3be6ac2c7a718257ec3aff333",
+         "platform": {
+            "architecture": "ppc64le",
+            "os": "linux"
+         }
+      },
+   ]
+}
+```
+
+### 2) (Recommended as a Best Practice) Ensure that node affinity expressions are set to match the supported platforms
+
+Kubernetes provides a mechanism called [nodeAffinity][node-affinity] which can be used to limit the possible node 
+targets where a pod can be scheduled. This is especially important to ensure correct scheduling behavior in clusters 
+with nodes that span across multiple platforms (i.e. heterogeneous clusters).
+
+**Kubernetes manifest example**
+
+```yaml
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/arch
+          operator: In
+          values:
+          - amd64
+          - arm64
+          - ppc64le
+          - s390x
+      - matchExpressions:
+          - key: kubernetes.io/os
+            operator: In
+            values:
+              - linux
+```
+
+**Golang Example**
+
+```go 
+Template: corev1.PodTemplateSpec{
+    ...
+    Spec: corev1.PodSpec{
+        Affinity: &corev1.Affinity{
+            NodeAffinity: &corev1.NodeAffinity{
+                RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
+                    NodeSelectorTerms: []corev1.NodeSelectorTerm{
+                        {
+                            MatchExpressions: []corev1.NodeSelectorRequirement{
+                                {
+                                    Key:      "kubernetes.io/arch",
+                                    Operator: "In",
+                                    Values:   []string{"amd64"},
+                                },
+                                {
+                                    Key:      "kubernetes.io/os",
+                                    Operator: "In",
+                                    Values:   []string{"linux"},
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+        },
+        SecurityContext: &corev1.PodSecurityContext{
+            ...
+        },
+        Containers: []corev1.Container{{
+            ...
+        }},
+    },
+```
+
+<aside class="note">
+<h1> Example(s) </h1>
+
+You can look for some code examples by checking the code which is generated via the Deploy 
+Image plugin. ([More info](../plugins/deploy-image-plugin-v1-alpha.md))
+
+</aside>
+
+## Producing projects that support multiple platforms
+
+You can use `docker buildx` to cross-compile via emulation ([QEMU](https://www.qemu.org/)) to build the manager image.
```

Comment:
It might be nice to include a link to the official Docker Buildx documentation:
```suggestion
You can use [`docker buildx`][buildx] to cross-compile via emulation ([QEMU](https://www.qemu.org/)) to build the manager image.
```